optimization/generate_profile.py

```diff
@@ -105,38 +102,51 @@ def load_course_map(course_name="COTA"):
             pitch = numpy.arctan(float(clean_data[i][1]) / 10)
             elev_profile.append(pitch)
     if course_name == "ASC":
-       pass 
+        pass
     return elev_profile
  
 if __name__ == '__main__':
     elev_profile = load_course_map()
     # Load in the distance and necessary time for a lap
     distance = 5490
-    time = 420 # max allowable time in s
-    iterations = 500  # Number of iterations to use
+    time = 420
+    iterations = 500 # Number of iterations to use
     init_profile = generate_initial_profile(time, distance, elev_profile)
     v_profile = init_profile
+    last_real_solution = v_profile
     values = []
     # Calculate the fitness of each population:
     values.append(calculate_fit(init_profile, elev_profile))
     min_value = 1000000
     min_time = 1000000
+    max_race_time = 1000
```

Comment:
This should be user-given

Reply:
How should we implement user-given values? I imagine its not just input(). Would we want to read from a separate conifg file? Not sure what the standard is.